composer.json

```
@@ -12,20 +12,20 @@
         },
         {
             "name": "Symfony Community",
-            "homepage": "http://symfony.com/contributors"
+            "homepage": "https://symfony.com/contributors"
         }
     ],
     "require": {
-        "php": ">=5.6",
-        "symfony/monolog-bridge": "~3.4 || ~4.0 || ^5.0",
+        "php": ">=7.1.3",
+        "symfony/monolog-bridge": "~4.0 || ^5.0",
```

I think we can be bold here.

```suggestion
        "symfony/monolog-bridge": "~4.4 || ^5.0",
```

The feature set of 3.4 and 4.0 is equivalent, so we won't gain much from dropping 3.4 alone.

---

agreed. And 3.4 still receive security fixes while 4.3 already stopped a while ago, so there is no reason to keep support 4.3 once we drop 3.4